Examples/SpatialObjects/SpatialObjectTransforms.cxx

```diff
@@ -22,53 +22,65 @@
 // transformations and the Object and World "spaces" associated with a spatial
 // object.
 //
-// Object Space
-//
-// SpatialObjects have one primary coordinate space that is readily available
-// to them, their ObjectSpace. This is the space in which the object was
+// \begin{description}
+// \item[Object Space]. SpatialObjects have one primary coordinate space
+// that is readily available
+// to them, their \code{ObjectSpace}. This is the space in which the object was
 // inherently defined. No transforms are applied to the points/values that
 // get/set into this space. All children of an object are added into this space.
 //
-// ObjectToParentTransform
-//
-// SpatialObjects have only one transform that they directly control, their
-// ObjectToParentTransform. This transform specifies how an object's space is
-// transformed to fit into its parent's ObjectSpace. The
-// ObjectToParentTransform is an affine transform, and it is confirmed to be
-// invertible when assigned, or the assignment fails.
-//
-// WorldSpace
+// \item[ObjectToParentTransform]. SpatialObjects have only one transform
+// that they directly control, their
+// \code{ObjectToParentTransform}. This transform specifies how an object's
+// \code{ObjectSpace} is
+// transformed to fit into its parent's \code{ObjectSpace}. The
+// \code{ObjectToParentTransform} is an affine transform, and it is confirmed
+// to be invertible when assigned, or the assignment fails.
 //
-// WorldSpace is not directly controlled by any SpatialObject except the
+// \item[WorldSpace]. \code{WorldSpace} is not directly controlled by any
+// SpatialObject except the
 // SpatialObject at the top level of the parent-child tree hierarchy of
 // Spatial Objects. That is, any SpatialObject that does not have a parent
-// exists in a WorldSpace that is defined by applying its
-// ObjectToParentTransform to its ObjectSpace.
+// exists in a \code{WorldSpace} that is defined by applying its
+// \code{ObjectToParentTransform} to its \code{ObjectSpace}.
+//
+// \end{description}
 //
 // Several member functions and variables are available to every SpatialObject
 // so that they can readily access the WorldSpace in which they exist:
 //
-// * ProtectedComputeObjectToWorldTransform: Composes its ObjectToParentTransform
-// with its parent's cached ObjectToObjectToWorldTransform, to determine the
-// transform from the object's ObjectSpace to WorldSpace.   This transform is
+// \begin{description}
+//
+// \item[ProtectedComputeObjectToWorldTransform()]: This function is called
```

Comment:
ProtectedComputeObjectToWorldTransform - wasn't this method renamed?

Reply:
I wish - but I decided not to fight that battle :)    It is named "ProtectedComputeObjectToWorldTransform()"...for now...